.github/workflows/docs-firebase-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Firebase Deploy Gcloud
+
+on:
+  push: 
+    branches: 
+      - main
+      - simulate-simnode
+
+env:
+  FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2     
+      
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.4.0'
+      
+      - name: Install Rust
+        run: |
+         rustup set profile minimal
+         rustup toolchain install stable
+         rustup default stable
+
+      - name: Install mdbook
+        run: |
+          mkdir bin
+          curl -sSL https://github.com/rust-lang/mdBook/releases/download/v0.4.14/mdbook-v0.4.14-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=bin
+          echo "$(pwd)/bin" >> $GITHUB_PATH
```

Comment:
We have/should have a nix step for this; we never manually install rust in CI, as that will lead to different versions.